bal/blockchain.py

```diff
@@ -119,47 +132,89 @@ def do_new_transaction():
+            get_private_from_wallet(),
+            blockchain.get_unspent_tx_outs(),
+            blockchain.transaction_pool.get_transaction_pool())
         return jsonify(True), 200
     except Exception as e:
         traceback.print_exc()
         tx = str(e)
         return jsonify(False), 200
 
+
 @app.route('/chain', methods=['GET'])
 def do_full_chain():
     return jsonify(blockchain.full_chain()), 200
 
+
 @app.route('/chain/length', methods=['GET'])
 def do_chain_length():
     return jsonify(blockchain.full_chain()['length']), 200
 
+
 @app.route('/peers', methods=['GET'])
 def do_get_peers():
     return jsonify(blockchain.p2p.get_peers()), 200
 
+
 @app.route('/peers/register', methods=['POST'])
 def do_register_peers():
     values = yaml.safe_load(json.dumps(request.get_json()))['peer']
     if not values:
         return 'Missing values', 200
     return jsonify(blockchain.p2p.add_peer(values)), 200
 
+
 if __name__ == '__main__':
     from argparse import ArgumentParser
 
     parser = ArgumentParser()
-    parser.add_argument('-p', '--port', default=5000, type=int, help='port to listen on')
-    parser.add_argument('-s', '--socket', default=6001, type=int, help='p2p port to listen on')
+    parser.add_argument(
+        '-p',
+        '--port',
+        default=5000,
+        type=int,
+        help='port to listen on')
+    parser.add_argument(
+        '-s',
+        '--socket',
+        default=6001,
+        type=int,
+        help='p2p port to listen on')
     parser.add_argument('-db', '--database', default='', help='db file')
-    parser.add_argument('-v', '--variant', default='pow', help='variant of blockchain "pow" or "pos"')
-    parser.add_argument('-d', '--difficulty', default=4, type=int, help='initial difficulty')
-    parser.add_argument('-k', '--keystore', default='/tmp/private_key.pem', help='where the keystore located. default: private_key.pem')
-    parser.add_argument('-sp', '--simulationpath', default='', help='specifies if it is a simulation run and where simulation logs will be kept.')
-    parser.add_argument('-n', '--name', default='bc', help='specifies blockchain node name(mostly for simulations)')
+    parser.add_argument(
+        '-v',
+        '--variant',
+        default='pow',
+        help='variant of blockchain "pow" or "pos"')
+    parser.add_argument(
+        '-d',
+        '--difficulty',
+        default=4,
+        type=int,
+        help='initial difficulty')
+    parser.add_argument(
+        '-k',
+        '--keystore',
+        default='/tmp/private_key.pem',
+        help='where the keystore located. default: private_key.pem')
+    parser.add_argument(
+        '-sp',
+        '--simulationpath',
+        default='',
+        help='specifies if it is a simulation run and where simulation logs will be kept.')
+    parser.add_argument(
+        '-n',
+        '--name',
+        default='bc',
+        help='specifies blockchain node name(mostly for simulations)')
```

Comment:
same concern here. These lines are too long. Could they be more compact? (if they're too long just split them by `help` parameter maybe?